.jenkins/win-build.sh

```diff
@@ -32,8 +32,9 @@ cat >ci_scripts/build_pytorch.bat <<EOL
 set PATH=C:\\Program Files\\CMake\\bin;C:\\Program Files\\7-Zip;C:\\curl-7.57.0-win64-mingw\\bin;C:\\Program Files\\Git\\cmd;C:\\Program Files\\Amazon\\AWSCLI;%PATH%
 
 :: Install MKL
-aws s3 cp s3://ossci-windows/mkl.7z mkl.7z --quiet && 7z x -aoa mkl.7z -omkl
-set LIB=%cd%\\mkl;%LIB%
+aws s3 cp s3://ossci-windows/mkl_with_headers.7z mkl.7z --quiet && 7z x -aoa mkl.7z -omkl
+set CMAKE_INCLUDE_PATH=%cd%\\mkl\\include
```

Comment:
What's going on here? Shouldn't the cmake be able to figure this out?

Reply:
Since the directory is not in environment PATH, we need to tell CMAKE where to look for.

Reply:
@SsnL Is there a reason that we cannot use mkl-include here just like Unix systems?